ANTLR.md

```diff
@@ -25,6 +25,6 @@ To build ANTLR4's golang listener or visitor, go to the directory where you
 have your antlr4 grammar `.g4` file. And then execute the following command.
 
 ```bash
-antlr -Dlanguage=Go -o parser -package grulev2 -lib . -listener -visitor grulev2.g4
+antlr -Dlanguage=Go -o parser -package grulev3 -lib . -listener -visitor grulev3.g4
```

Comment:
@niallnsec, @newm4n   do we need to generate using `-listener -visitor` ? because i didnt use those options and didnt find any meaningful difference

Reply:
Well, grule is actually using the listener pattern. I think `-listener` option still needed while `-visitor` we don't. I don't know if the `-listener` is the default here.